Source/OCMock/OCMockObject.m

```diff
@@ -31,15 +31,80 @@
 #import "OCMExpectationRecorder.h"
 #import "OCMQuantifier.h"
 
+@class XCTestCase;
+
+// gCurrentMocksToStopRecorder will point to either gTestCaseMocksToStop or gTestSuiteMocksToStop
+// depending on whether we are currently recoring testcase mocks or test suite mocks.
+// Controlled by OCMockXCTestObserver.
+static NSHashTable<OCMockObject *> *gTestCaseMocksToStop;
+static NSHashTable<OCMockObject *> *gTestSuiteMocksToStop;
+static NSHashTable<OCMockObject *> *gCurrentMocksToStopRecorder;
```

Comment:
As general feedback, why have all these as static variables in this file? Wouldn't it be possible to move the state and logic to a separate class? It could have a singleton (shared instance) to manage the state, right?

Reply:
Will do if that's preferred.